06-loki-functors.md

```diff
@@ -153,7 +160,36 @@ mass == mass_child
 ```
 
 The usage of LoKi functors extends much further than in the interactive `GaudiPython` world.
-On one side, they constitute the basis of particle filtering in the *selection framework*;
-on the other, they can be used directly inside our `DaVinci` jobs to store specific bits of information in our ntuples without the need for a complicated C++-based algorithms.
-This second option will be discussed in the [TupleTools and branches lesson](12-add-tupletools.html).
+On one side, they constitute the basis of particle filtering in the *selection framework*, discussed in the [Building your own decay chain](06-building-decays.html) lesson.
```

Comment:
However one should only use "One the one hand" if you are going to follow it up with "and on the other ..". So I suggest removing it here.